cli/tests/integrations/test_marathon_pods.py

```diff
@@ -0,0 +1,173 @@
+import contextlib
+import json
+import re
+
+from .common import exec_command, file_bytes, file_json
+
+FILE_PATH_BASE = 'tests/data/marathon/pods/'
+
+GOOD_POD_ID = 'good-pod'
+GOOD_POD_FILE_PATH = FILE_PATH_BASE + 'good.json'
+UPDATED_GOOD_POD_FILE_PATH = FILE_PATH_BASE + 'updated_good.json'
+
+DOUBLE_POD_ID = 'double-pod'
+DOUBLE_POD_FILE_PATH = FILE_PATH_BASE + 'double.json'
+
+TRIPLE_POD_ID = 'winston'
+TRIPLE_POD_FILE_PATH = FILE_PATH_BASE + 'doubleplusgood.json'
+
+
+def test_pod_add_from_file_then_remove():
+    exit_status = _pod_add_from_file(GOOD_POD_FILE_PATH)
+    assert exit_status == 0
+
+    exit_status = _pod_remove(GOOD_POD_ID, force=False)
+    assert exit_status == 0
+
+
+def test_pod_add_from_stdin_then_force_remove():
```

Comment:
Could you be more specific? I need to add a pod anyway in order to test removal.